### PR TITLE
Als 2110

### PR DESCRIPTION
**Pull request by enyambudzi**

### CODE REVIEW

File: scripts/install_cloudwatch.ps1 [hunk 1]
```diff
@@ -2,10 +2,10 @@
 #--------------------------------------------------------------------------
 # create temp folder for downloading amazon cloudwatch agent
 #--------------------------------------------------------------------------
-New-Item c:\temp -ItemType Directory -ErrorAction Ignore
-Invoke-WebRequest -Uri 'https://s3.amazonaws.com/amazoncloudwatch-agent/windows/amd64/latest/amazon-cloudwatch-agent.msi' -OutFile 'c:\temp\amazon-cloudwatch-agent.msi'
+#New-Item c:\temp -ItemType Directory -ErrorAction Ignore
+Invoke-WebRequest -Uri 'https://s3.amazonaws.com/amazoncloudwatch-agent/windows/amd64/latest/amazon-cloudwatch-agent.msi' -OutFile 'c:\cloudwatch_installer\amazon-cloudwatch-agent.msi'
 #--------------------------------------------------------------------------
-Start-Process msiexec.exe -Wait -ArgumentList '/i c:\temp\amazon-cloudwatch-agent.msi'
+Start-Process msiexec.exe -Wait -ArgumentList '/i c:\cloudwatch_installer\amazon-cloudwatch-agent.msi'
```
Comment (ChrisMKinsella) [inline scripts/install_cloudwatch.ps1:8]: is the /i argument ok for unattended install or do you need to specifiy /q (quiet) ? - sometimes /q is implicit depending upon other args

Comment (enyambudzi) [reply scripts/install_cloudwatch.ps1:8]: This script is currently used by way of attended installation. Can be looked at when we start the automation work

Comment (ChrisMKinsella) [reply scripts/install_cloudwatch.ps1:8]: ok

**Review by ChrisMKinsella — APPROVED** [review]
LGTM

**Pull request merged**